customers.views WARNING 2024-06-13 11:37:11,917 views 7577 139653644453440 1
customers.views WARNING 2024-06-13 11:37:11,917 views 7577 139653644453440 GET
customers.views WARNING 2024-06-13 11:37:11,917 views 7577 139653644453440 2
customers.views WARNING 2024-06-13 11:37:11,917 views 7577 139653644453440 /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/8/invoice_create/
customers.views WARNING 2024-06-13 11:37:11,917 views 7577 139653644453440 3
customers.views WARNING 2024-06-13 11:37:11,918 views 7577 139653644453440 b''
customers.views WARNING 2024-06-13 11:37:11,918 views 7577 139653644453440 4
customers.views WARNING 2024-06-13 11:37:11,918 views 7577 139653644453440 <QueryDict: {}>
customers.views WARNING 2024-06-13 11:37:11,918 views 7577 139653644453440 5
customers.views WARNING 2024-06-13 11:37:11,918 views 7577 139653644453440 <QueryDict: {}>
customers.views WARNING 2024-06-13 11:37:11,918 views 7577 139653644453440 6
customers.views WARNING 2024-06-13 11:37:11,918 views 7577 139653644453440 {}
customers.views WARNING 2024-06-13 11:37:12,143 views 7577 139653644453440 7
customers.views WARNING 2024-06-13 11:37:12,144 views 7577 139653644453440 Order object (8)
customers.views WARNING 2024-06-13 11:37:12,144 views 7577 139653644453440 8
customers.views WARNING 2024-06-13 11:37:12,144 views 7577 139653644453440 <QueryDict: {}>
customers.views WARNING 2024-06-13 11:37:12,144 views 7577 139653644453440 []
customers.views WARNING 2024-06-13 11:37:12,144 views 7577 139653644453440 9
customers.views WARNING 2024-06-13 11:37:12,144 views 7577 139653644453440 <QueryDict: {}>
customers.views WARNING 2024-06-13 11:37:12,144 views 7577 139653644453440 []
customers.views WARNING 2024-06-13 11:37:12,144 views 7577 139653644453440 10
customers.views WARNING 2024-06-13 11:37:12,144 views 7577 139653644453440 {'amount_paid': <django.forms.fields.DecimalField object at 0x7f03a5f0e7b0>, 'note': <django.forms.fields.CharField object at 0x7f03a5f2e9c0>}
customers.views WARNING 2024-06-13 11:37:12,144 views 7577 139653644453440 11
customers.views WARNING 2024-06-13 11:37:12,144 views 7577 139653644453440 Invoice object (None)
urllib3.connectionpool DEBUG 2024-06-13 11:37:12,522 connectionpool 7577 139653636044352 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-06-13 11:37:12,575 connectionpool 7577 139653636044352 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 200 None
django.request WARNING 2024-06-13 11:37:13,178 log 7577 139653386585664 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.utils.autoreload INFO 2024-06-13 11:39:34,594 autoreload 7577 139653709634624 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 11:39:35,123 autoreload 7707 140063804161088 Watching for file changes with StatReloader
urllib3.connectionpool DEBUG 2024-06-13 11:39:37,429 connectionpool 7707 140063738979904 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-06-13 11:39:37,475 connectionpool 7707 140063738979904 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 200 4117
django.request WARNING 2024-06-13 11:39:38,165 log 7707 140063488853568 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
customers.views WARNING 2024-06-13 11:39:42,758 views 7707 140063480460864 1
customers.views WARNING 2024-06-13 11:39:42,758 views 7707 140063480460864 GET
customers.views WARNING 2024-06-13 11:39:42,758 views 7707 140063480460864 2
customers.views WARNING 2024-06-13 11:39:42,758 views 7707 140063480460864 /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/8/invoice_create/
customers.views WARNING 2024-06-13 11:39:42,758 views 7707 140063480460864 3
customers.views WARNING 2024-06-13 11:39:42,758 views 7707 140063480460864 b''
customers.views WARNING 2024-06-13 11:39:42,758 views 7707 140063480460864 4
customers.views WARNING 2024-06-13 11:39:42,758 views 7707 140063480460864 <QueryDict: {}>
customers.views WARNING 2024-06-13 11:39:42,758 views 7707 140063480460864 5
customers.views WARNING 2024-06-13 11:39:42,759 views 7707 140063480460864 <QueryDict: {}>
customers.views WARNING 2024-06-13 11:39:42,759 views 7707 140063480460864 6
customers.views WARNING 2024-06-13 11:39:42,759 views 7707 140063480460864 {}
customers.views WARNING 2024-06-13 11:39:42,982 views 7707 140063480460864 7
customers.views WARNING 2024-06-13 11:39:42,983 views 7707 140063480460864 Order object (8)
customers.views WARNING 2024-06-13 11:39:42,983 views 7707 140063480460864 8
customers.views WARNING 2024-06-13 11:39:42,983 views 7707 140063480460864 <QueryDict: {}>
customers.views WARNING 2024-06-13 11:39:42,983 views 7707 140063480460864 []
customers.views WARNING 2024-06-13 11:39:42,983 views 7707 140063480460864 9
customers.views WARNING 2024-06-13 11:39:42,983 views 7707 140063480460864 <QueryDict: {}>
customers.views WARNING 2024-06-13 11:39:42,983 views 7707 140063480460864 []
customers.views WARNING 2024-06-13 11:39:42,983 views 7707 140063480460864 10
customers.views WARNING 2024-06-13 11:39:42,983 views 7707 140063480460864 {'amount_paid': <django.forms.fields.DecimalField object at 0x7f632172f7a0>, 'note': <django.forms.fields.CharField object at 0x7f632172dfd0>}
customers.views WARNING 2024-06-13 11:39:42,983 views 7707 140063480460864 <built-in method values of dict object at 0x7f63216b2d80>
customers.views WARNING 2024-06-13 11:39:42,983 views 7707 140063480460864 11
customers.views WARNING 2024-06-13 11:39:42,983 views 7707 140063480460864 Invoice object (None)
django.request WARNING 2024-06-13 11:39:43,714 log 7707 140063463675456 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.utils.autoreload INFO 2024-06-13 11:41:43,700 autoreload 7707 140063804161088 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 11:41:44,233 autoreload 7831 140364707929152 Watching for file changes with StatReloader
customers.views WARNING 2024-06-13 11:41:48,315 views 7831 140364642747968 1
customers.views WARNING 2024-06-13 11:41:48,315 views 7831 140364642747968 GET
customers.views WARNING 2024-06-13 11:41:48,315 views 7831 140364642747968 2
customers.views WARNING 2024-06-13 11:41:48,315 views 7831 140364642747968 /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/8/invoice_create/
customers.views WARNING 2024-06-13 11:41:48,315 views 7831 140364642747968 3
customers.views WARNING 2024-06-13 11:41:48,315 views 7831 140364642747968 b''
customers.views WARNING 2024-06-13 11:41:48,315 views 7831 140364642747968 4
customers.views WARNING 2024-06-13 11:41:48,315 views 7831 140364642747968 <QueryDict: {}>
customers.views WARNING 2024-06-13 11:41:48,315 views 7831 140364642747968 5
customers.views WARNING 2024-06-13 11:41:48,315 views 7831 140364642747968 <QueryDict: {}>
customers.views WARNING 2024-06-13 11:41:48,315 views 7831 140364642747968 6
customers.views WARNING 2024-06-13 11:41:48,315 views 7831 140364642747968 {}
customers.views WARNING 2024-06-13 11:41:48,574 views 7831 140364642747968 7
customers.views WARNING 2024-06-13 11:41:48,574 views 7831 140364642747968 Order object (8)
customers.views WARNING 2024-06-13 11:41:48,574 views 7831 140364642747968 8
customers.views WARNING 2024-06-13 11:41:48,574 views 7831 140364642747968 <QueryDict: {}>
customers.views WARNING 2024-06-13 11:41:48,574 views 7831 140364642747968 []
customers.views WARNING 2024-06-13 11:41:48,574 views 7831 140364642747968 9
customers.views WARNING 2024-06-13 11:41:48,574 views 7831 140364642747968 <QueryDict: {}>
customers.views WARNING 2024-06-13 11:41:48,574 views 7831 140364642747968 []
customers.views WARNING 2024-06-13 11:41:48,575 views 7831 140364642747968 10
customers.views WARNING 2024-06-13 11:41:48,575 views 7831 140364642747968 {'amount_paid': <django.forms.fields.DecimalField object at 0x7fa930b520f0>, 'note': <django.forms.fields.CharField object at 0x7fa930b52150>}
customers.views WARNING 2024-06-13 11:41:48,575 views 7831 140364642747968 <built-in method values of dict object at 0x7fa930aed040>
customers.views WARNING 2024-06-13 11:41:48,575 views 7831 140364642747968 11
customers.views WARNING 2024-06-13 11:41:48,575 views 7831 140364642747968 Invoice object (None)
customers.views WARNING 2024-06-13 11:41:48,575 views 7831 140364642747968 12
customers.views WARNING 2024-06-13 11:41:48,575 views 7831 140364642747968 {'amount_paid': <django.forms.fields.DecimalField object at 0x7fa930af8530>, 'note': <django.forms.fields.CharField object at 0x7fa930af99d0>}
customers.views WARNING 2024-06-13 11:41:48,575 views 7831 140364642747968 <built-in method values of dict object at 0x7fa930aef200>
customers.views WARNING 2024-06-13 11:41:48,575 views 7831 140364642747968 13
customers.views WARNING 2024-06-13 11:41:48,575 views 7831 140364642747968 Invoice object (None)
urllib3.connectionpool DEBUG 2024-06-13 11:41:48,889 connectionpool 7831 140364564395584 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-06-13 11:41:48,945 connectionpool 7831 140364564395584 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 200 4117
django.request WARNING 2024-06-13 11:41:49,388 log 7831 140364547610176 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.utils.autoreload INFO 2024-06-13 11:47:44,670 autoreload 7831 140364707929152 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 11:47:45,198 autoreload 8298 139952405261376 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-13 11:49:13,913 autoreload 8298 139952405261376 /workspace/django-renterprise/customers/urls.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 11:49:14,447 autoreload 8406 139825720695872 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-13 11:49:29,912 autoreload 8406 139825720695872 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 11:49:30,447 autoreload 8452 140126071897152 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-13 11:52:23,570 autoreload 8452 140126071897152 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 11:52:24,096 autoreload 8663 139962304646208 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-13 11:52:57,900 autoreload 8663 139962304646208 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 11:52:58,460 autoreload 8809 140034436856896 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-13 12:10:12,944 autoreload 8809 140034436856896 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 12:10:13,447 autoreload 9627 140334920789056 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-13 12:10:33,999 autoreload 9627 140334920789056 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 12:10:34,528 autoreload 9661 140440852188224 Watching for file changes with StatReloader
urllib3.connectionpool DEBUG 2024-06-13 12:10:49,210 connectionpool 9661 140440787007040 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-06-13 12:10:49,294 connectionpool 9661 140440787007040 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 200 4117
django.request WARNING 2024-06-13 12:10:49,990 log 9661 140440743913024 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:11:11,959 log 9661 140440221230656 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:12:11,129 log 9661 140439692752448 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:13:33,190 log 9661 140439164274240 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:13:35,906 log 9661 140439155881536 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/8/invoice_create/
django.request WARNING 2024-06-13 12:13:35,952 log 9661 140439147488832 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:13:36,142 log 9661 140439139096128 Not Found: /favicon.ico
django.request WARNING 2024-06-13 12:13:36,153 log 9661 140438652581440 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/8/invoice_create/styles.css.map
django.request WARNING 2024-06-13 12:13:42,824 log 9661 140438644188736 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/8/invoice_create/styles.css.map
django.request WARNING 2024-06-13 12:13:43,054 log 9661 140438635796032 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request ERROR 2024-06-13 12:13:51,512 log 9661 140438627403328 Internal Server Error: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/8/invoice_create/22/test/
Traceback (most recent call last):
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/django-renterprise/customers/views.py", line 365, in invoice_create
    amount_paid=Decimal(amount_paid),
                ^^^^^^^
NameError: name 'Decimal' is not defined
django.request WARNING 2024-06-13 12:13:51,577 log 9661 140438619010624 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:13:51,753 log 9661 140438610617920 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/8/invoice_create/22/test/styles.css.map
django.utils.autoreload INFO 2024-06-13 12:14:54,308 autoreload 9661 140440852188224 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 12:14:54,858 autoreload 9940 140311752735808 Watching for file changes with StatReloader
django.request WARNING 2024-06-13 12:14:56,093 log 9940 140311615501888 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/8/invoice_create/22/test/styles.css.map
django.request WARNING 2024-06-13 12:14:56,347 log 9940 140311607109184 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
urllib3.connectionpool DEBUG 2024-06-13 12:14:58,045 connectionpool 9940 140311598716480 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-06-13 12:14:58,183 connectionpool 9940 140311598716480 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 200 4117
django.request WARNING 2024-06-13 12:14:58,774 log 9940 140311078630976 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request ERROR 2024-06-13 12:15:02,400 log 9940 140311070238272 Internal Server Error: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/8/invoice_create/22/Test/
Traceback (most recent call last):
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/django-renterprise/customers/views.py", line 366, in invoice_create
    amount_paid=Decimal(amount_paid),
                ^^^^^^^
NameError: name 'Decimal' is not defined. Did you mean: 'decimal'?
django.request WARNING 2024-06-13 12:15:02,473 log 9940 140311061845568 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:15:02,729 log 9940 140311045060160 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/8/invoice_create/22/Test/styles.css.map
django.request WARNING 2024-06-13 12:15:02,730 log 9940 140311053452864 Not Found: /favicon.ico
django.utils.autoreload INFO 2024-06-13 12:15:34,032 autoreload 9940 140311752735808 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 12:15:34,578 autoreload 9998 140347419515968 Watching for file changes with StatReloader
django.request WARNING 2024-06-13 12:15:36,223 log 9998 140347354334784 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
urllib3.connectionpool DEBUG 2024-06-13 12:15:38,288 connectionpool 9998 140347344877120 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-06-13 12:15:38,352 connectionpool 9998 140347344877120 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 200 4117
django.request WARNING 2024-06-13 12:15:38,641 log 9998 140347336468032 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:15:38,949 log 9998 140346956695104 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request ERROR 2024-06-13 12:15:49,505 log 9998 140346948302400 Internal Server Error: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/8/invoice_create/23/Test/
Traceback (most recent call last):
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/django-renterprise/customers/views.py", line 362, in invoice_create
    invoice = Invoice(
              ^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/db/models/base.py", line 543, in __init__
    _setattr(self, field.name, rel_obj)
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/db/models/fields/related_descriptors.py", line 266, in __set__
    raise ValueError(
ValueError: Cannot assign "8": "Invoice.order" must be a "Order" instance.
django.request WARNING 2024-06-13 12:15:49,584 log 9998 140346939909696 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:15:49,795 log 9998 140346931516992 Not Found: /favicon.ico
django.request WARNING 2024-06-13 12:15:49,811 log 9998 140346445002304 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/8/invoice_create/23/Test/styles.css.map
django.utils.autoreload INFO 2024-06-13 12:16:27,873 autoreload 9998 140347419515968 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 12:16:28,412 autoreload 10089 140569549581376 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-13 12:16:32,586 autoreload 10089 140569549581376 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 12:16:33,108 autoreload 10102 140482536447040 Watching for file changes with StatReloader
django.request WARNING 2024-06-13 12:16:35,617 log 10102 140482403366464 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/8/invoice_create/23/Test/styles.css.map
django.request WARNING 2024-06-13 12:16:35,855 log 10102 140482394973760 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
urllib3.connectionpool DEBUG 2024-06-13 12:16:37,003 connectionpool 10102 140482385532480 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-06-13 12:16:37,062 connectionpool 10102 140482385532480 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 200 None
django.request WARNING 2024-06-13 12:16:37,670 log 10102 140481870689856 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:16:44,053 log 10102 140481845511744 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:16:47,214 log 10102 140481828726336 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:16:47,489 log 10102 140481820333632 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:16:50,185 log 10102 140481325426240 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:16:50,566 log 10102 140481317033536 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:17:02,473 log 10102 140481308640832 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/8/invoice_create/
django.request WARNING 2024-06-13 12:17:02,519 log 10102 140481300248128 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:17:02,639 log 10102 140481291855424 Not Found: /favicon.ico
django.request WARNING 2024-06-13 12:17:02,649 log 10102 140481283462720 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/8/invoice_create/styles.css.map
django.request WARNING 2024-06-13 12:17:06,914 log 10102 140480796948032 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/8/invoice_create/styles.css.map
django.request WARNING 2024-06-13 12:17:07,138 log 10102 140480788555328 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:17:39,061 log 10102 140480780162624 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/8/invoice_create/
django.request WARNING 2024-06-13 12:17:39,111 log 10102 140480771769920 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:17:39,245 log 10102 140480763377216 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/8/invoice_create/styles.css.map
django.request WARNING 2024-06-13 12:19:32,241 log 10102 140480754984512 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/8/invoice_create/styles.css.map
django.request WARNING 2024-06-13 12:19:32,485 log 10102 140480746591808 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:19:33,889 log 10102 140480218113600 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:19:38,778 log 10102 140479714813504 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:19:38,953 log 10102 140479698028096 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:19:43,512 log 10102 140479689635392 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/8/invoice_create/
django.request WARNING 2024-06-13 12:19:43,565 log 10102 140479681242688 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:19:43,683 log 10102 140479672849984 Not Found: /favicon.ico
django.request WARNING 2024-06-13 12:19:43,692 log 10102 140479186335296 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/8/invoice_create/styles.css.map
django.request WARNING 2024-06-13 12:19:45,213 log 10102 140479169549888 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/8/invoice_create/styles.css.map
django.request WARNING 2024-06-13 12:19:45,489 log 10102 140479161157184 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:40:14,564 log 10102 140478632678976 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:40:28,669 log 10102 140478599108160 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:40:43,712 log 10102 140478087415360 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:40:49,489 log 10102 140478070629952 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:40:49,716 log 10102 140477575722560 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:40:51,516 log 10102 140477558937152 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:40:51,668 log 10102 140477550544448 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:41:58,251 log 10102 140477525366336 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:41:58,419 log 10102 140477038851648 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:42:03,529 log 10102 140476988495424 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:42:13,216 log 10102 140476485195328 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:42:13,384 log 10102 140476468409920 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:42:15,247 log 10102 140476451624512 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:42:15,418 log 10102 140475956717120 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:42:17,452 log 10102 140475931539008 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:42:17,599 log 10102 140475923146304 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:45:57,808 log 10102 140475394668096 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:46:29,688 log 10102 140474866189888 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:46:45,942 log 10102 140474337711680 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:47:24,959 log 10102 140474304140864 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:47:25,184 log 10102 140473817626176 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:47:29,904 log 10102 140473800840768 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:47:30,222 log 10102 140473784055360 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:47:50,638 log 10102 140473272362560 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:47:50,798 log 10102 140473263969856 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:47:55,794 log 10102 140473247184448 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:47:56,034 log 10102 140473238791744 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.utils.autoreload INFO 2024-06-13 12:50:05,483 autoreload 12032 139840879955008 Watching for file changes with StatReloader
urllib3.connectionpool DEBUG 2024-06-13 12:50:07,102 connectionpool 12032 139840814773824 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-06-13 12:50:07,166 connectionpool 12032 139840814773824 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 200 None
django.request WARNING 2024-06-13 12:51:00,755 log 12032 139839890511424 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:52:01,712 log 12032 139839353640512 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:52:13,048 log 12032 139839328462400 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:52:13,454 log 12032 139839320069696 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/8/order_notes_list/styles.css.map
django.request WARNING 2024-06-13 12:52:14,427 log 12032 139838816769600 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/8/order_notes_list/styles.css.map
django.request WARNING 2024-06-13 12:52:14,905 log 12032 139838263113280 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/8/order_notes_list/create/styles.css.map
django.request WARNING 2024-06-13 12:53:43,220 log 12032 139837180978752 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/8/order_notes_list/create/styles.css.map
django.request WARNING 2024-06-13 12:53:45,973 log 12032 139836669285952 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/8/order_notes_list/create/styles.css.map
django.request WARNING 2024-06-13 12:53:46,325 log 12032 139836660893248 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/8/order_notes_list/styles.css.map
django.request WARNING 2024-06-13 12:53:47,640 log 12032 139836635715136 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/8/order_notes_list/styles.css.map
django.request WARNING 2024-06-13 12:53:47,813 log 12032 139836140807744 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:54:27,101 log 12032 139835612329536 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:55:24,532 log 12032 139835561973312 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:55:51,858 log 12032 139835033495104 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:56:55,180 log 12032 139834530195008 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 12:56:55,380 log 12032 139834521802304 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/8/order_notes_list/styles.css.map
django.request WARNING 2024-06-13 13:21:01,565 log 12032 139834505016896 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/8/order_notes_list/styles.css.map
django.request WARNING 2024-06-13 13:21:22,985 log 12032 139834496624192 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/8/order_notes_list/styles.css.map
django.request WARNING 2024-06-13 13:24:10,379 log 12032 139833984931392 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/8/order_notes_list/styles.css.map
django.request WARNING 2024-06-13 13:24:10,744 log 12032 139833968145984 Not Found: /styles.css.map
django.request WARNING 2024-06-13 13:24:11,844 log 12032 139833414489664 Not Found: /styles.css.map
django.request WARNING 2024-06-13 13:24:12,015 log 12032 139832902796864 Not Found: /favicon.ico
django.request WARNING 2024-06-13 13:24:12,306 log 12032 139832894404160 Not Found: /admin/styles.css.map
django.request WARNING 2024-06-13 13:24:17,753 log 12032 139832391104064 Not Found: /styles.css.map
django.request WARNING 2024-06-13 13:24:17,778 log 12032 139832877618752 Not Found: /admin/styles.css.map
django.request WARNING 2024-06-13 13:24:19,427 log 12032 139832365925952 Not Found: /styles.css.map
django.request WARNING 2024-06-13 13:24:19,600 log 12032 139832357533248 Not Found: /items/styles.css.map
django.request WARNING 2024-06-13 13:24:21,743 log 12032 139832340747840 Not Found: /items/styles.css.map
django.request WARNING 2024-06-13 13:24:21,943 log 12032 139831854233152 Not Found: /items/create/item_type/styles.css.map
django.utils.autoreload INFO 2024-06-13 13:25:59,048 autoreload 12032 139840879955008 /workspace/django-renterprise/items/models.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 13:25:59,624 autoreload 13549 140662189374528 Watching for file changes with StatReloader
urllib3.connectionpool DEBUG 2024-06-13 13:26:00,716 connectionpool 13549 140662057989696 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-06-13 13:26:00,777 connectionpool 13549 140662057989696 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 200 None
django.request WARNING 2024-06-13 13:26:01,481 log 13549 140662022321728 Not Found: /items/create/item_type/styles.css.map
django.utils.autoreload INFO 2024-06-13 13:28:12,793 autoreload 14146 140173952724032 Watching for file changes with StatReloader
urllib3.connectionpool DEBUG 2024-06-13 13:28:18,815 connectionpool 14146 140173887542848 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-06-13 13:28:18,892 connectionpool 14146 140173887542848 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 200 None
django.request WARNING 2024-06-13 13:28:19,762 log 14146 140173869676096 Not Found: /favicon.ico
django.utils.autoreload INFO 2024-06-13 13:29:00,929 autoreload 14146 140173952724032 /workspace/django-renterprise/customers/models.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 13:29:01,473 autoreload 14228 140240439327808 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-13 13:29:34,238 autoreload 14228 140240439327808 /workspace/django-renterprise/customers/models.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 13:29:34,767 autoreload 14288 140220593896512 Watching for file changes with StatReloader
urllib3.connectionpool DEBUG 2024-06-13 13:29:37,402 connectionpool 14288 140220528715328 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-06-13 13:29:37,473 connectionpool 14288 140220528715328 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 200 4117
django.request WARNING 2024-06-13 13:29:39,258 log 14288 140217931527744 Not Found: /favicon.ico
django.utils.autoreload INFO 2024-06-13 13:32:23,050 autoreload 14288 140220593896512 /workspace/django-renterprise/orders/admin.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 13:32:23,638 autoreload 14493 140485654129728 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-13 13:32:41,106 autoreload 14493 140485654129728 /workspace/django-renterprise/orders/admin.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 13:32:41,623 autoreload 14518 139738648939584 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-13 13:32:54,014 autoreload 14518 139738648939584 /workspace/django-renterprise/orders/admin.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 13:32:54,557 autoreload 14543 140537388076096 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-13 13:33:27,280 autoreload 14543 140537388076096 /workspace/django-renterprise/orders/models.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 13:33:27,817 autoreload 14585 140395675860032 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-13 13:33:35,123 autoreload 14585 140395675860032 /workspace/django-renterprise/orders/models.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 13:33:35,656 autoreload 14602 139951916854336 Watching for file changes with StatReloader
urllib3.connectionpool DEBUG 2024-06-13 13:33:44,192 connectionpool 14602 139951777773120 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-06-13 13:33:44,254 connectionpool 14602 139951777773120 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 200 None
django.request WARNING 2024-06-13 13:33:46,259 log 14602 139950150374976 Not Found: /favicon.ico
django.utils.autoreload INFO 2024-06-13 13:34:39,144 autoreload 14602 139951916854336 /workspace/django-renterprise/customers/models.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 13:34:39,686 autoreload 14702 140579303943232 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-13 13:34:50,075 autoreload 14702 140579303943232 /workspace/django-renterprise/customers/models.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 13:34:50,605 autoreload 14729 139917191134272 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-13 13:35:25,404 autoreload 14729 139917191134272 /workspace/django-renterprise/orders/models.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 13:35:25,959 autoreload 14802 140049105738816 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-13 13:35:29,230 autoreload 14802 140049105738816 /workspace/django-renterprise/orders/models.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 13:35:29,769 autoreload 14811 139885197958208 Watching for file changes with StatReloader
urllib3.connectionpool DEBUG 2024-06-13 13:35:35,587 connectionpool 14811 139885132777024 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-06-13 13:35:35,654 connectionpool 14811 139885132777024 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 200 None
django.request WARNING 2024-06-13 13:35:36,607 log 14811 139882882115136 Not Found: /favicon.ico
django.utils.autoreload INFO 2024-06-13 13:35:41,252 autoreload 14811 139885197958208 /workspace/django-renterprise/orders/models.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 13:35:41,793 autoreload 14869 140173965388864 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-13 13:37:16,622 autoreload 14869 140173965388864 /workspace/django-renterprise/orders/models.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 13:37:17,129 autoreload 15054 140218884455488 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-13 13:37:55,527 autoreload 15054 140218884455488 /workspace/django-renterprise/orders/models.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 13:37:56,026 autoreload 15105 140172394458176 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-13 13:38:25,373 autoreload 15105 140172394458176 /workspace/django-renterprise/orders/models.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 13:38:25,902 autoreload 15141 140368698113088 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-13 13:38:32,107 autoreload 15141 140368698113088 /workspace/django-renterprise/orders/models.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 13:38:32,625 autoreload 15152 140626739559488 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-13 13:38:48,093 autoreload 15152 140626739559488 /workspace/django-renterprise/orders/models.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 13:38:48,619 autoreload 15212 139646465223744 Watching for file changes with StatReloader
urllib3.connectionpool DEBUG 2024-06-13 13:38:52,818 connectionpool 15212 139646400042560 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-06-13 13:38:52,894 connectionpool 15212 139646400042560 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 200 None
django.request WARNING 2024-06-13 13:38:54,600 log 15212 139643941021248 Not Found: /favicon.ico
django.utils.autoreload INFO 2024-06-13 13:39:08,239 autoreload 15212 139646465223744 /workspace/django-renterprise/orders/models.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 13:39:08,771 autoreload 15276 140286658681920 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-13 13:39:11,018 autoreload 15276 140286658681920 /workspace/django-renterprise/orders/models.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 13:39:11,548 autoreload 15283 140293924084800 Watching for file changes with StatReloader
urllib3.connectionpool DEBUG 2024-06-13 13:39:27,335 connectionpool 15283 140293858903616 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-06-13 13:39:27,408 connectionpool 15283 140293858903616 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 200 4117
django.request WARNING 2024-06-13 13:39:28,661 log 15283 140291398948416 Not Found: /favicon.ico
django.utils.autoreload INFO 2024-06-13 13:44:57,815 autoreload 15283 140293924084800 /workspace/django-renterprise/orders/models.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 13:44:58,387 autoreload 15720 140305129221184 Watching for file changes with StatReloader
urllib3.connectionpool DEBUG 2024-06-13 13:45:07,801 connectionpool 15720 140305064040000 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-06-13 13:45:07,864 connectionpool 15720 140305064040000 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 200 None
django.request WARNING 2024-06-13 13:45:10,713 log 15720 140303176537664 Not Found: /favicon.ico
django.utils.autoreload INFO 2024-06-13 13:46:23,247 autoreload 15720 140305129221184 /workspace/django-renterprise/orders/models.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 13:46:23,813 autoreload 15898 140611066141760 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-13 13:46:38,266 autoreload 15898 140611066141760 /workspace/django-renterprise/orders/admin.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 13:46:38,806 autoreload 15924 140668304432192 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-13 13:46:44,110 autoreload 15924 140668304432192 /workspace/django-renterprise/orders/admin.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 13:46:44,625 autoreload 15951 139940177206336 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-13 13:47:04,178 autoreload 15951 139940177206336 /workspace/django-renterprise/orders/models.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 13:47:04,702 autoreload 16013 140319116921920 Watching for file changes with StatReloader
urllib3.connectionpool DEBUG 2024-06-13 13:47:11,363 connectionpool 16013 140319051740736 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-06-13 13:47:11,419 connectionpool 16013 140319051740736 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 200 None
django.request ERROR 2024-06-13 13:47:11,553 log 16013 140319051740736 Internal Server Error: /admin/orders/invoice/5/change/
Traceback (most recent call last):
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/response.py", line 92, in rendered_content
    return template.render(context, self._request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader_tags.py", line 208, in render
    return template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 177, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/defaulttags.py", line 321, in render
    return nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/defaulttags.py", line 321, in render
    return nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1070, in render
    return render_value_in_context(output, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1047, in render_value_in_context
    value = str(value)
            ^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/utils/html.py", line 420, in <lambda>
    klass.__str__ = lambda self: mark_safe(klass_str(self))
                                           ^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/forms/boundfield.py", line 34, in __str__
    return self.as_widget()
           ^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/forms/boundfield.py", line 107, in as_widget
    return widget.render(
           ^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/forms/widgets.py", line 280, in render
    context = self.get_context(name, value, attrs)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/contrib/admin/widgets.py", line 320, in get_context
    "rendered_widget": self.widget.render(name, value, attrs),
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/forms/widgets.py", line 280, in render
    context = self.get_context(name, value, attrs)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/forms/widgets.py", line 758, in get_context
    context = super().get_context(name, value, attrs)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/forms/widgets.py", line 717, in get_context
    context["widget"]["optgroups"] = self.optgroups(
                                     ^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/forms/widgets.py", line 657, in optgroups
    for index, (option_value, option_label) in enumerate(self.choices):
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/forms/models.py", line 1404, in __iter__
    yield self.choice(obj)
          ^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/forms/models.py", line 1418, in choice
    self.field.label_from_instance(obj),
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/forms/models.py", line 1504, in label_from_instance
    return str(obj)
           ^^^^^^^^
  File "/workspace/django-renterprise/orders/models.py", line 38, in __str__
    return f"Order ID : {self.id} - {self.custommer.last_name}"
                                     ^^^^^^^^^^^^^^
AttributeError: 'Order' object has no attribute 'custommer'. Did you mean: 'customer'?
django.request WARNING 2024-06-13 13:47:11,884 log 16013 140319039395392 Not Found: /favicon.ico
django.utils.autoreload INFO 2024-06-13 13:47:17,263 autoreload 16013 140319116921920 /workspace/django-renterprise/orders/models.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 13:47:17,803 autoreload 16041 140099205966912 Watching for file changes with StatReloader
urllib3.connectionpool DEBUG 2024-06-13 13:47:20,565 connectionpool 16041 140099140785728 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-06-13 13:47:20,640 connectionpool 16041 140099140785728 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 200 4117
django.request WARNING 2024-06-13 13:47:22,004 log 16041 140096950998592 Not Found: /favicon.ico
django.request WARNING 2024-06-13 13:48:05,113 log 16041 140093738165824 Not Found: /items/create/item_type/styles.css.map
django.request WARNING 2024-06-13 13:48:05,453 log 16041 140093243258432 Not Found: /styles.css.map
django.request WARNING 2024-06-13 13:48:06,265 log 16041 140093226473024 Not Found: /styles.css.map
django.request WARNING 2024-06-13 13:48:06,474 log 16041 140093218080320 Not Found: /customers/styles.css.map
django.request WARNING 2024-06-13 13:48:09,002 log 16041 140093201294912 Not Found: /customers/styles.css.map
django.request WARNING 2024-06-13 13:48:09,230 log 16041 140093192902208 Not Found: /styles.css.map
django.request WARNING 2024-06-13 13:48:10,695 log 16041 140092697994816 Not Found: /styles.css.map
django.request WARNING 2024-06-13 13:48:11,019 log 16041 140092689602112 Not Found: /orders/styles.css.map
django.request WARNING 2024-06-13 13:48:16,169 log 16041 140092672816704 Not Found: /orders/styles.css.map
django.request WARNING 2024-06-13 13:48:16,360 log 16041 140092656031296 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 13:48:19,329 log 16041 140092161123904 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-13 13:48:19,626 log 16041 140092152731200 Not Found: /styles.css.map
django.utils.autoreload INFO 2024-06-13 13:49:20,571 autoreload 16727 140292926938176 Watching for file changes with StatReloader
urllib3.connectionpool DEBUG 2024-06-13 13:49:23,653 connectionpool 16727 140292835481152 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-06-13 13:49:23,729 connectionpool 16727 140292835481152 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 200 None
django.request WARNING 2024-06-13 13:49:28,590 log 16727 140292455904832 Not Found: /favicon.ico
django.utils.autoreload INFO 2024-06-13 13:51:41,350 autoreload 16727 140292926938176 /workspace/django-renterprise/items/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 13:51:41,885 autoreload 16962 140552563698752 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-13 13:51:52,152 autoreload 16962 140552563698752 /workspace/django-renterprise/items/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 13:51:52,678 autoreload 16988 139814967901248 Watching for file changes with StatReloader
urllib3.connectionpool DEBUG 2024-06-13 13:52:06,661 connectionpool 16988 139814902720064 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-06-13 13:52:06,729 connectionpool 16988 139814902720064 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 200 None
django.request WARNING 2024-06-13 13:52:07,149 log 16988 139814885901888 Not Found: /styles.css.map
django.request WARNING 2024-06-13 13:52:07,387 log 16988 139814674363968 Not Found: /items/styles.css.map
django.request WARNING 2024-06-13 13:52:10,740 log 16988 139814649185856 Not Found: /items/styles.css.map
django.request WARNING 2024-06-13 13:52:11,079 log 16988 139814640793152 Not Found: /items/23/styles.css.map
django.request ERROR 2024-06-13 13:52:16,614 log 16988 139814631351872 Internal Server Error: /items/23/status_change/4
Traceback (most recent call last):
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/django-renterprise/items/views.py", line 98, in item_status_change
    if status == REPAIR or status == "Repair":
                 ^^^^^^
NameError: name 'REPAIR' is not defined
django.request WARNING 2024-06-13 13:52:16,664 log 16988 139814622959168 Not Found: /items/23/styles.css.map
django.request WARNING 2024-06-13 13:52:17,014 log 16988 139814070384192 Not Found: /items/23/status_change/styles.css.map
django.utils.autoreload INFO 2024-06-13 13:52:44,850 autoreload 16988 139814967901248 /workspace/django-renterprise/items/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 13:52:45,413 autoreload 17077 140511571319872 Watching for file changes with StatReloader
django.request WARNING 2024-06-13 13:52:48,817 log 17077 140511506138688 Not Found: /items/23/status_change/styles.css.map
django.request WARNING 2024-06-13 13:52:49,031 log 17077 140511497729600 Not Found: /items/23/styles.css.map
urllib3.connectionpool DEBUG 2024-06-13 13:52:50,534 connectionpool 17077 140511488271936 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-06-13 13:52:50,610 connectionpool 17077 140511488271936 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 200 None
django.request WARNING 2024-06-13 13:52:51,045 log 17077 140511104976448 Not Found: /items/23/styles.css.map
django.request ERROR 2024-06-13 13:52:54,857 log 17077 140511096583744 Internal Server Error: /items/23/status_change/4
Traceback (most recent call last):
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/django-renterprise/items/views.py", line 99, in item_status_change
    item.repair_date = datetime.utcnow()
                       ^^^^^^^^
NameError: name 'datetime' is not defined. Did you forget to import 'datetime'
django.request WARNING 2024-06-13 13:52:54,935 log 17077 140511088191040 Not Found: /items/23/styles.css.map
django.request WARNING 2024-06-13 13:52:55,248 log 17077 140511079798336 Not Found: /items/23/status_change/styles.css.map
django.request WARNING 2024-06-13 13:52:55,261 log 17077 140510593283648 Not Found: /favicon.ico
django.utils.autoreload INFO 2024-06-13 13:53:11,208 autoreload 17077 140511571319872 /workspace/django-renterprise/items/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 13:53:11,782 autoreload 17136 140656477551680 Watching for file changes with StatReloader
urllib3.connectionpool DEBUG 2024-06-13 13:53:17,637 connectionpool 17136 140656412370496 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-06-13 13:53:17,732 connectionpool 17136 140656412370496 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 200 None
django.request WARNING 2024-06-13 13:53:17,891 log 17136 140656386094656 Not Found: /items/23/styles.css.map
django.request WARNING 2024-06-13 13:53:18,104 log 17136 140656369276480 Not Found: /items/23/styles.css.map
django.request WARNING 2024-06-13 13:53:23,670 log 17136 140655858796096 Not Found: /items/23/styles.css.map
django.request WARNING 2024-06-13 13:53:30,985 log 17136 140655842010688 Not Found: /items/23/styles.css.map
django.request WARNING 2024-06-13 13:53:31,188 log 17136 140655833617984 Not Found: /items/styles.css.map
django.request WARNING 2024-06-13 13:53:39,044 log 17136 140655338710592 Not Found: /items/styles.css.map
django.request WARNING 2024-06-13 13:53:39,371 log 17136 140655330317888 Not Found: /items/17/styles.css.map
django.request WARNING 2024-06-13 13:53:43,197 log 17136 140655313532480 Not Found: /items/17/styles.css.map
django.request WARNING 2024-06-13 13:53:43,398 log 17136 140655305139776 Not Found: /items/styles.css.map
django.request WARNING 2024-06-13 13:54:00,429 log 17136 140654810232384 Not Found: /items/styles.css.map
django.request WARNING 2024-06-13 13:54:00,598 log 17136 140654801839680 Not Found: /items/styles.css.map
django.request WARNING 2024-06-13 13:54:02,831 log 17136 140654785054272 Not Found: /items/styles.css.map
django.request WARNING 2024-06-13 13:54:03,039 log 17136 140654776661568 Not Found: /items/styles.css.map
django.request WARNING 2024-06-13 13:54:09,415 log 17136 140654256576064 Not Found: /items/styles.css.map
django.request WARNING 2024-06-13 13:54:12,350 log 17136 140654239790656 Not Found: /items/styles.css.map
django.request WARNING 2024-06-13 13:54:12,698 log 17136 140654231397952 Not Found: /items/17/styles.css.map
django.request WARNING 2024-06-13 13:54:17,071 log 17136 140653736490560 Not Found: /items/17/styles.css.map
django.request WARNING 2024-06-13 13:54:17,284 log 17136 140653728097856 Not Found: /items/styles.css.map
django.request WARNING 2024-06-13 13:54:18,603 log 17136 140653711312448 Not Found: /items/styles.css.map
django.request WARNING 2024-06-13 13:54:18,822 log 17136 140653702919744 Not Found: /items/28/styles.css.map
django.request WARNING 2024-06-13 13:54:21,542 log 17136 140653686134336 Not Found: /items/28/styles.css.map
django.request WARNING 2024-06-13 13:54:21,746 log 17136 140653199619648 Not Found: /items/styles.css.map
django.request WARNING 2024-06-13 13:54:23,698 log 17136 140653182834240 Not Found: /items/styles.css.map
django.request WARNING 2024-06-13 13:54:23,918 log 17136 140653174441536 Not Found: /items/28/styles.css.map
django.request WARNING 2024-06-13 13:54:27,119 log 17136 140653149263424 Not Found: /items/28/styles.css.map
django.request WARNING 2024-06-13 13:54:30,483 log 17136 140652654356032 Not Found: /items/28/styles.css.map
django.request WARNING 2024-06-13 13:54:30,916 log 17136 140652645963328 Not Found: /items/styles.css.map
django.request WARNING 2024-06-13 13:54:35,720 log 17136 140652629177920 Not Found: /items/styles.css.map
django.request WARNING 2024-06-13 13:54:36,030 log 17136 140652620785216 Not Found: /items/28/styles.css.map
django.request WARNING 2024-06-13 13:54:37,998 log 17136 140652125877824 Not Found: /items/28/styles.css.map
django.request WARNING 2024-06-13 13:54:38,218 log 17136 140652117485120 Not Found: /items/styles.css.map
django.request WARNING 2024-06-13 13:54:47,745 log 17136 140652075521600 Not Found: /items/styles.css.map
django.utils.autoreload INFO 2024-06-13 13:55:17,247 autoreload 17136 140656477551680 /workspace/django-renterprise/items/models.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 13:55:17,788 autoreload 17329 140049004354624 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-13 13:55:24,081 autoreload 17329 140049004354624 /workspace/django-renterprise/items/models.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 13:55:24,595 autoreload 17340 140157673802816 Watching for file changes with StatReloader
urllib3.connectionpool DEBUG 2024-06-13 13:55:27,323 connectionpool 17340 140157533550144 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-06-13 13:55:27,401 connectionpool 17340 140157533550144 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 200 None
django.request WARNING 2024-06-13 13:55:28,030 log 17340 140157499979328 Not Found: /items/styles.css.map
django.request WARNING 2024-06-13 13:55:35,801 log 17340 140157483193920 Not Found: /items/styles.css.map
django.request WARNING 2024-06-13 13:55:36,019 log 17340 140156996679232 Not Found: /items/styles.css.map
django.request WARNING 2024-06-13 13:55:45,349 log 17340 140156979893824 Not Found: /items/styles.css.map
django.request WARNING 2024-06-13 13:55:45,512 log 17340 140156971501120 Not Found: /items/styles.css.map
django.utils.autoreload INFO 2024-06-13 13:56:18,816 autoreload 17340 140157673802816 /workspace/django-renterprise/items/models.py changed, reloading.
django.utils.autoreload INFO 2024-06-13 13:56:19,365 autoreload 17468 140142159154240 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-13 13:57:15,089 autoreload 17939 139625473379392 Watching for file changes with StatReloader
urllib3.connectionpool DEBUG 2024-06-13 13:57:16,520 connectionpool 17939 139625408198208 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-06-13 13:57:16,597 connectionpool 17939 139625408198208 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 200 None
django.request WARNING 2024-06-13 13:57:17,056 log 17939 139625398740544 Not Found: /items/create/styles.css.map
django.request WARNING 2024-06-13 13:57:17,395 log 17939 139625389282880 Not Found: /items/create/styles.css.map
django.utils.autoreload INFO 2024-06-13 15:01:11,048 autoreload 43672 140521057365056 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 00:59:49,962 autoreload 7443 140160241728576 Watching for file changes with StatReloader
urllib3.connectionpool DEBUG 2024-06-14 00:59:59,798 connectionpool 7443 140160176547392 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-06-14 00:59:59,884 connectionpool 7443 140160176547392 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 200 4117
django.request WARNING 2024-06-14 01:00:00,289 log 7443 140159948420672 Not Found: /items/create/styles.css.map
django.request WARNING 2024-06-14 01:00:00,539 log 7443 140159931635264 Not Found: /styles.css.map
django.request WARNING 2024-06-14 01:00:01,763 log 7443 140159914849856 Not Found: /styles.css.map
django.request WARNING 2024-06-14 01:00:02,044 log 7443 140159906457152 Not Found: /customers/styles.css.map
django.request WARNING 2024-06-14 01:00:03,987 log 7443 140159337096768 Not Found: /customers/styles.css.map
django.request WARNING 2024-06-14 01:00:04,159 log 7443 140159328704064 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/styles.css.map
django.request WARNING 2024-06-14 01:00:06,324 log 7443 140159311918656 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/styles.css.map
django.request WARNING 2024-06-14 01:00:06,663 log 7443 140159303525952 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/customer_notes_list/styles.css.map
django.utils.autoreload INFO 2024-06-14 01:00:14,742 autoreload 7443 140160241728576 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:00:15,231 autoreload 7573 140716025416768 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:00:24,592 autoreload 7573 140716025416768 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:00:25,074 autoreload 7628 140556617301056 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:01:11,035 autoreload 7628 140556617301056 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:01:11,568 autoreload 7823 139828697259072 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:01:57,521 autoreload 7823 139828697259072 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:01:58,010 autoreload 7899 140544288889920 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:02:03,380 autoreload 7899 140544288889920 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:02:03,868 autoreload 7930 140351068793920 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:02:11,202 autoreload 7930 140351068793920 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:02:11,689 autoreload 7978 140060323114048 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:02:13,931 autoreload 7978 140060323114048 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:02:14,417 autoreload 8000 140568238353472 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:02:34,962 autoreload 8000 140568238353472 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:02:35,453 autoreload 8047 140353951349824 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:02:44,807 autoreload 8047 140353951349824 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:02:45,301 autoreload 8098 140596458978368 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:02:48,562 autoreload 8098 140596458978368 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:02:49,082 autoreload 8127 139621104573504 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:03:12,666 autoreload 8127 139621104573504 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:03:13,152 autoreload 8239 139885112376384 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:03:30,646 autoreload 8239 139885112376384 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:03:31,133 autoreload 8358 139966223232064 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:03:58,790 autoreload 8358 139966223232064 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:03:59,274 autoreload 8442 140206564228160 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:04:40,157 autoreload 8442 140206564228160 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:04:40,642 autoreload 8640 139921704598592 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:04:59,161 autoreload 8640 139921704598592 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:04:59,646 autoreload 8697 139867925388352 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:05:03,919 autoreload 8697 139867925388352 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:05:04,409 autoreload 8730 139870883343424 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:05:38,164 autoreload 8730 139870883343424 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:05:38,650 autoreload 8797 140136192203840 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:06:41,887 autoreload 8797 140136192203840 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:06:42,376 autoreload 9071 139826244189248 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:06:48,687 autoreload 9071 139826244189248 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:06:49,172 autoreload 9112 140575374830656 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:07:22,928 autoreload 9112 140575374830656 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:07:23,416 autoreload 9282 140535648918592 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:07:34,802 autoreload 9282 140535648918592 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:07:35,292 autoreload 9335 139709208947776 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:07:59,899 autoreload 9335 139709208947776 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:08:00,384 autoreload 9411 140413135313984 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:08:16,852 autoreload 9411 140413135313984 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:08:17,342 autoreload 9467 139825419533376 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:08:34,828 autoreload 9467 139825419533376 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:08:35,313 autoreload 9521 139838596717632 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:09:09,064 autoreload 9521 139838596717632 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:09:09,549 autoreload 9588 140700500520000 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:09:18,910 autoreload 9588 140700500520000 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:09:19,393 autoreload 9643 140138538024000 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:09:24,682 autoreload 9643 140138538024000 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:09:25,209 autoreload 9673 140623758404672 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:09:42,713 autoreload 9673 140623758404672 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:09:43,206 autoreload 9735 140618725923904 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:11:07,804 autoreload 9735 140618725923904 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:11:08,290 autoreload 10162 140649788468288 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:11:40,013 autoreload 10162 140649788468288 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:11:40,504 autoreload 10293 140701417434176 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:11:50,876 autoreload 10293 140701417434176 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:11:51,363 autoreload 10350 140434490702912 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:11:55,639 autoreload 10350 140434490702912 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:11:56,129 autoreload 10383 139906218056768 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:12:58,372 autoreload 10383 139906218056768 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:12:58,860 autoreload 10491 140383868726336 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:13:05,171 autoreload 10491 140383868726336 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:13:05,660 autoreload 10532 140106181209152 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:13:12,990 autoreload 10532 140106181209152 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:13:13,481 autoreload 10572 140118357605440 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:13:29,955 autoreload 10572 140118357605440 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:13:30,439 autoreload 10628 139633495907392 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:13:55,040 autoreload 10628 139633495907392 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:13:55,528 autoreload 10695 139823710514240 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:14:07,935 autoreload 10695 139823710514240 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:14:08,419 autoreload 10750 140097635900480 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:14:34,038 autoreload 10750 140097635900480 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:14:34,527 autoreload 10830 139768185855040 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:14:55,073 autoreload 10830 139768185855040 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:14:55,564 autoreload 10886 140120387411008 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:14:58,821 autoreload 10886 140120387411008 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:14:59,304 autoreload 10915 140040402586688 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:15:07,647 autoreload 10915 140040402586688 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:15:08,133 autoreload 10948 140266108222528 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:31:07,097 autoreload 10948 140266108222528 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:31:07,588 autoreload 11828 140395672529984 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:37:34,221 autoreload 11828 140395672529984 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:37:34,709 autoreload 12229 140494834658368 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:38:22,724 autoreload 12229 140494834658368 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:38:23,210 autoreload 12310 140672014130240 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:38:39,676 autoreload 12310 140672014130240 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:38:40,161 autoreload 12377 140049094757440 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:40:05,818 autoreload 12377 140049094757440 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:40:06,309 autoreload 12596 140406389734464 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:40:09,564 autoreload 12596 140406389734464 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:40:10,065 autoreload 12626 139765289286720 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:40:21,454 autoreload 12626 139765289286720 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:40:21,939 autoreload 12658 139984021548096 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:40:44,513 autoreload 12658 139984021548096 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:40:45,003 autoreload 12719 139706015673408 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:43:41,098 autoreload 12719 139706015673408 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:43:41,586 autoreload 13148 139988944243776 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:43:49,926 autoreload 13148 139988944243776 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:43:50,414 autoreload 13201 140565378665536 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:43:52,655 autoreload 13201 140565378665536 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:43:53,137 autoreload 13223 140486520024128 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:44:03,510 autoreload 13223 140486520024128 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:44:03,990 autoreload 13262 140543939259456 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:46:31,624 autoreload 13262 140543939259456 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:46:32,115 autoreload 13732 139663752879168 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:46:50,625 autoreload 13732 139663752879168 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:46:51,113 autoreload 13865 140666311994432 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:47:01,483 autoreload 13865 140666311994432 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:47:01,975 autoreload 13945 140667662801984 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:47:29,629 autoreload 13945 140667662801984 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:47:30,112 autoreload 14082 140671238691904 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:47:40,484 autoreload 14082 140671238691904 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:47:40,973 autoreload 14135 140654254242880 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:47:48,297 autoreload 14135 140654254242880 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:47:48,779 autoreload 14188 139757875092544 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:47:55,083 autoreload 14188 139757875092544 /workspace/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:47:55,568 autoreload 14219 140495004654656 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:48:03,906 autoreload 14219 140495004654656 /workspace/django-renterprise/customers/models.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:48:04,397 autoreload 14272 139998411082816 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 01:48:10,705 autoreload 14272 139998411082816 /workspace/django-renterprise/customers/urls.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 01:48:11,189 autoreload 14304 140574864292928 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 11:36:41,679 autoreload 2680 140096613551168 Watching for file changes with StatReloader
urllib3.connectionpool DEBUG 2024-06-14 11:36:44,443 connectionpool 2680 140096548226624 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-06-14 11:36:44,653 connectionpool 2680 140096548226624 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 200 4117
django.request WARNING 2024-06-14 11:36:44,847 log 2680 140096447698496 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/customer_notes_list/styles.css.map
django.request WARNING 2024-06-14 11:36:45,026 log 2680 140096430913088 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/customer_notes_list/styles.css.map
django.request WARNING 2024-06-14 11:39:23,638 log 2680 140095843718720 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/customer_notes_list/styles.css.map
django.request WARNING 2024-06-14 11:47:24,173 log 2680 140095818540608 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/customer_notes_list/styles.css.map
django.request WARNING 2024-06-14 11:47:24,331 log 2680 140095810147904 Not Found: /styles.css.map
django.request WARNING 2024-06-14 11:49:07,031 log 2680 140095306847808 Not Found: /styles.css.map
django.request WARNING 2024-06-14 11:49:07,213 log 2680 140095298455104 Not Found: /items/styles.css.map
django.request WARNING 2024-06-14 11:49:08,390 log 2680 140095281669696 Not Found: /items/styles.css.map
django.request WARNING 2024-06-14 11:49:08,642 log 2680 140095273276992 Not Found: /items/create/item_type/styles.css.map
django.request WARNING 2024-06-14 11:49:20,820 log 2680 140094753191488 Not Found: /items/create/item_type/styles.css.map
django.request WARNING 2024-06-14 11:49:52,929 log 2680 140094233105984 Not Found: /items/create/item_type/styles.css.map
django.request WARNING 2024-06-14 11:51:36,508 log 2680 140094207927872 Not Found: /items/create/item_type/styles.css.map
django.request WARNING 2024-06-14 11:51:36,851 log 2680 140094199535168 Not Found: /styles.css.map
django.request WARNING 2024-06-14 11:51:37,905 log 2680 140093704627776 Not Found: /styles.css.map
django.request WARNING 2024-06-14 11:51:38,127 log 2680 140093696235072 Not Found: /items/styles.css.map
django.request WARNING 2024-06-14 11:51:41,761 log 2680 140093679449664 Not Found: /items/styles.css.map
django.request WARNING 2024-06-14 11:51:42,000 log 2680 140093671056960 Not Found: /styles.css.map
django.request WARNING 2024-06-14 11:52:04,793 log 2680 140093150971456 Not Found: /styles.css.map
django.utils.autoreload INFO 2024-06-14 12:20:34,267 autoreload 9774 139630023265344 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 12:24:35,344 autoreload 9774 139630023265344 /workspace/django-renterprise/menu/urls.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 12:24:35,828 autoreload 9981 139691043167296 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 13:27:42,083 autoreload 9981 139691043167296 /workspace/django-renterprise/menu/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 13:27:42,573 autoreload 12626 140299285240896 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 13:28:19,327 autoreload 12626 140299285240896 /workspace/django-renterprise/menu/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 13:28:19,823 autoreload 12711 139897214491712 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 13:28:41,341 autoreload 12711 139897214491712 /workspace/django-renterprise/menu/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 13:28:41,833 autoreload 12753 140016042863680 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 13:28:53,191 autoreload 12753 140016042863680 /workspace/django-renterprise/menu/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 13:28:53,690 autoreload 12768 139779674958912 Watching for file changes with StatReloader
urllib3.connectionpool DEBUG 2024-06-14 13:28:59,143 connectionpool 12768 139779609634368 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-06-14 13:28:59,207 connectionpool 12768 139779609634368 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 200 None
django.request WARNING 2024-06-14 13:28:59,566 log 12768 139779592816192 Not Found: /styles.css.map
django.request WARNING 2024-06-14 13:29:00,214 log 12768 139779375101504 Not Found: /styles.css.map
django.utils.autoreload INFO 2024-06-14 13:29:32,613 autoreload 12768 139779674958912 /workspace/django-renterprise/menu/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 13:29:33,110 autoreload 12811 140312402291776 Watching for file changes with StatReloader
urllib3.connectionpool DEBUG 2024-06-14 13:30:20,778 connectionpool 12811 140312336967232 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-06-14 13:30:20,828 connectionpool 12811 140312336967232 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 200 None
django.request WARNING 2024-06-14 13:30:21,465 log 12811 140312302282304 Not Found: /styles.css.map
django.utils.autoreload INFO 2024-06-14 13:30:36,475 autoreload 12811 140312402291776 /workspace/django-renterprise/menu/urls.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 13:30:36,971 autoreload 12872 139926610406464 Watching for file changes with StatReloader
urllib3.connectionpool DEBUG 2024-06-14 13:30:39,203 connectionpool 12872 139926477731392 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-06-14 13:30:39,310 connectionpool 12872 139926477731392 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 200 None
django.request WARNING 2024-06-14 13:30:39,836 log 12872 139926443112000 Not Found: /styles.css.map
django.request WARNING 2024-06-14 13:30:42,124 log 12872 139926426326592 Not Found: /styles.css.map
django.request WARNING 2024-06-14 13:30:44,203 log 12872 139925932467776 Not Found: /styles.css.map
django.request WARNING 2024-06-14 13:30:44,787 log 12872 139925924075072 Not Found: /items/styles.css.map
django.utils.autoreload INFO 2024-06-14 13:30:51,561 autoreload 12872 139926610406464 /workspace/django-renterprise/menu/urls.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 13:30:52,063 autoreload 12908 139758964976704 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 13:31:05,494 autoreload 12908 139758964976704 /workspace/django-renterprise/menu/urls.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 13:31:05,983 autoreload 12928 139632218602560 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 13:31:45,845 autoreload 12928 139632218602560 /workspace/django-renterprise/customers/urls.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 13:31:46,336 autoreload 12966 140480126204992 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 13:31:54,682 autoreload 12966 140480126204992 /workspace/django-renterprise/customers/urls.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 13:31:55,177 autoreload 12980 140501064617024 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 13:34:41,146 autoreload 12980 140501064617024 /workspace/django-renterprise/renterprise/urls.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 13:34:41,649 autoreload 13160 140027661691968 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 13:36:13,425 autoreload 13160 140027661691968 /workspace/django-renterprise/renterprise/settings.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 13:36:13,914 autoreload 13264 140017469828160 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 13:38:45,664 autoreload 13264 140017469828160 /workspace/django-renterprise/menu/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 13:38:46,160 autoreload 13524 139858102082624 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 13:39:02,643 autoreload 13524 139858102082624 /workspace/django-renterprise/menu/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 13:39:03,137 autoreload 13550 140662343945280 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 13:39:53,177 autoreload 13550 140662343945280 /workspace/django-renterprise/menu/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 13:39:53,678 autoreload 13592 140132684479552 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 13:39:58,974 autoreload 13592 140132684479552 /workspace/django-renterprise/menu/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 13:39:59,468 autoreload 13606 139894365162560 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 13:40:01,711 autoreload 13606 139894365162560 /workspace/django-renterprise/menu/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 13:40:02,205 autoreload 13618 140269860678720 Watching for file changes with StatReloader
urllib3.connectionpool DEBUG 2024-06-14 13:40:21,865 connectionpool 13618 140269795354176 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-06-14 13:40:21,930 connectionpool 13618 140269795354176 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 200 None
django.request WARNING 2024-06-14 13:40:22,303 log 13618 140269786945088 Not Found: /items/styles.css.map
django.request WARNING 2024-06-14 13:40:22,635 log 13618 140269777487424 Not Found: /styles.css.map
django.utils.autoreload INFO 2024-06-14 13:40:42,150 autoreload 13618 140269860678720 /workspace/django-renterprise/menu/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 13:40:42,647 autoreload 13658 140076927896640 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 13:41:07,262 autoreload 13658 140076927896640 /workspace/django-renterprise/menu/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 13:41:07,760 autoreload 13687 140280166251584 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 13:47:24,126 autoreload 13687 140280166251584 /workspace/django-renterprise/menu/urls.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 13:47:24,620 autoreload 14196 140069743828032 Watching for file changes with StatReloader
django.request WARNING 2024-06-14 13:47:26,805 log 14196 140069678503488 Not Found: /styles.css.map
urllib3.connectionpool DEBUG 2024-06-14 13:47:28,280 connectionpool 14196 140069669045824 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-06-14 13:47:28,382 connectionpool 14196 140069669045824 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 200 None
django.request WARNING 2024-06-14 13:47:28,768 log 14196 140069642769984 Not Found: /styles.css.map
django.request WARNING 2024-06-14 13:47:29,474 log 14196 140069151176256 Not Found: /styles.css.map
django.request WARNING 2024-06-14 13:47:33,093 log 14196 140068597519936 Not Found: /styles.css.map
django.request WARNING 2024-06-14 13:47:33,259 log 14196 140068563949120 Not Found: /favicon.ico
django.request WARNING 2024-06-14 13:47:33,549 log 14196 140068077434432 Not Found: /admin/styles.css.map
django.request WARNING 2024-06-14 13:47:34,177 log 14196 140068060649024 Not Found: /styles.css.map
django.request WARNING 2024-06-14 13:47:34,297 log 14196 140068069041728 Not Found: /admin/styles.css.map
django.request WARNING 2024-06-14 13:48:22,977 log 14196 140067540563520 Not Found: /styles.css.map
django.request WARNING 2024-06-14 13:48:24,520 log 14196 140067523778112 Not Found: /styles.css.map
django.request WARNING 2024-06-14 13:48:24,803 log 14196 140067515385408 Not Found: /customers/styles.css.map
django.request WARNING 2024-06-14 13:48:25,314 log 14196 140067506992704 Not Found: /customers/styles.css.map
django.request WARNING 2024-06-14 13:48:25,561 log 14196 140067498600000 Not Found: /styles.css.map
django.request WARNING 2024-06-14 13:48:26,298 log 14196 140067003692608 Not Found: /styles.css.map
django.request WARNING 2024-06-14 13:48:26,453 log 14196 140066995299904 Not Found: /favicon.ico
django.request WARNING 2024-06-14 13:48:26,733 log 14196 140066986907200 Not Found: /admin/styles.css.map
django.request WARNING 2024-06-14 13:48:27,361 log 14196 140066970121792 Not Found: /styles.css.map
django.request WARNING 2024-06-14 13:48:27,441 log 14196 140066978514496 Not Found: /admin/styles.css.map
django.request WARNING 2024-06-14 13:48:28,508 log 14196 140066953336384 Not Found: /styles.css.map
django.request WARNING 2024-06-14 13:48:28,772 log 14196 140066466821696 Not Found: /orders/styles.css.map
django.request WARNING 2024-06-14 13:48:29,672 log 14196 140066458428992 Not Found: /orders/styles.css.map
django.request WARNING 2024-06-14 13:48:30,014 log 14196 140066450036288 Not Found: /styles.css.map
django.request WARNING 2024-06-14 13:48:31,175 log 14196 140066433250880 Not Found: /styles.css.map
django.request WARNING 2024-06-14 13:48:31,602 log 14196 140066424858176 Not Found: /orders/styles.css.map
django.request WARNING 2024-06-14 13:48:31,790 log 14196 140066416465472 Not Found: /orders/styles.css.map
django.request WARNING 2024-06-14 13:48:31,966 log 14196 140065929950784 Not Found: /styles.css.map
django.request WARNING 2024-06-14 13:48:32,962 log 14196 140065913165376 Not Found: /styles.css.map
django.request WARNING 2024-06-14 13:48:33,311 log 14196 140065904772672 Not Found: /items/styles.css.map
django.request WARNING 2024-06-14 13:48:33,845 log 14196 140065896379968 Not Found: /items/styles.css.map
django.request WARNING 2024-06-14 13:48:34,166 log 14196 140065887987264 Not Found: /styles.css.map
django.utils.autoreload INFO 2024-06-14 22:20:36,500 autoreload 2369 140538501590080 Watching for file changes with StatReloader
urllib3.connectionpool DEBUG 2024-06-14 22:48:48,147 connectionpool 2369 140538436265536 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-06-14 22:48:48,304 connectionpool 2369 140538436265536 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 200 None
django.request WARNING 2024-06-14 22:48:49,026 log 2369 140538401580608 Not Found: /styles.css.map
django.request WARNING 2024-06-14 22:48:50,435 log 2369 140537906591296 Not Found: /styles.css.map
django.request WARNING 2024-06-14 22:48:50,684 log 2369 140537898198592 Not Found: /customers/styles.css.map
django.request WARNING 2024-06-14 22:48:52,678 log 2369 140537873020480 Not Found: /customers/styles.css.map
django.request WARNING 2024-06-14 22:48:52,837 log 2369 140537864627776 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/styles.css.map
django.request WARNING 2024-06-14 22:48:55,903 log 2369 140537369720384 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/styles.css.map
django.request WARNING 2024-06-14 22:48:56,274 log 2369 140537361327680 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-14 22:48:58,330 log 2369 140537344542272 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-14 22:48:58,783 log 2369 140537327756864 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-14 22:49:11,099 log 2369 140536824456768 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-14 22:49:11,339 log 2369 140536816064064 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-14 22:49:17,344 log 2369 140536799278656 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/orders/styles.css.map
django.request WARNING 2024-06-14 22:49:17,667 log 2369 140536790885952 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/styles.css.map
django.request WARNING 2024-06-14 22:49:22,070 log 2369 140536295978560 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/styles.css.map
django.request WARNING 2024-06-14 22:49:22,308 log 2369 140536279193152 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request WARNING 2024-06-14 22:50:57,762 log 2369 140535750714944 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request WARNING 2024-06-14 22:51:57,342 log 2369 140535222236736 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request WARNING 2024-06-14 22:53:24,258 log 2369 140535171880512 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request WARNING 2024-06-14 22:54:11,159 log 2369 140534643402304 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request WARNING 2024-06-14 22:55:58,100 log 2369 140534114924096 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request WARNING 2024-06-14 22:58:33,201 log 2369 140533586445888 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request WARNING 2024-06-14 22:59:29,655 log 2369 140533057967680 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request WARNING 2024-06-14 23:01:44,884 log 2369 140532529489472 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request WARNING 2024-06-14 23:02:55,444 log 2369 140532001011264 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request WARNING 2024-06-14 23:04:42,823 log 2369 140531950655040 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request WARNING 2024-06-14 23:07:18,352 log 2369 140531422176832 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request WARNING 2024-06-14 23:08:54,879 log 2369 140530893698624 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request WARNING 2024-06-14 23:17:36,570 log 2369 140530365220416 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request WARNING 2024-06-14 23:18:20,410 log 2369 140529836742208 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request WARNING 2024-06-14 23:19:21,630 log 2369 140529308264000 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request ERROR 2024-06-14 23:22:02,596 log 2369 140529299871296 Internal Server Error: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/
Traceback (most recent call last):
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/django-renterprise/orders/views.py", line 69, in order_create
    return render(
           ^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/defaulttags.py", line 321, in render
    return nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/urls/base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/urls/resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'customer_view' with keyword arguments '{'customer_token': ''}' not found. 1 pattern(s) tried: ['customers/(?P<customer_token>[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})/\\Z']
django.request WARNING 2024-06-14 23:22:02,913 log 2369 140529291478592 Not Found: /favicon.ico
django.request WARNING 2024-06-14 23:22:02,922 log 2369 140529283085888 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request ERROR 2024-06-14 23:22:46,936 log 2369 140529274693184 Internal Server Error: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/
Traceback (most recent call last):
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/django-renterprise/orders/views.py", line 69, in order_create
    return render(
           ^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/defaulttags.py", line 321, in render
    return nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/urls/base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/urls/resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'customer_view' with keyword arguments '{'customer_token': ''}' not found. 1 pattern(s) tried: ['customers/(?P<customer_token>[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})/\\Z']
django.request WARNING 2024-06-14 23:22:47,193 log 2369 140529266300480 Not Found: /favicon.ico
django.request WARNING 2024-06-14 23:22:47,254 log 2369 140528779785792 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request ERROR 2024-06-14 23:23:14,370 log 2369 140528771393088 Internal Server Error: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/
Traceback (most recent call last):
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/django-renterprise/orders/views.py", line 69, in order_create
    return render(
           ^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/backends/django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 200, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 513, in parse
    raise self.error(token, e)
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 511, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader_tags.py", line 293, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 513, in parse
    raise self.error(token, e)
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 511, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader_tags.py", line 232, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 513, in parse
    raise self.error(token, e)
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 511, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/defaulttags.py", line 953, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 513, in parse
    raise self.error(token, e)
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 511, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/defaulttags.py", line 1387, in url
    kwargs[name] = parser.compile_filter(value)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 600, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 703, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: '{{' from '{{'
django.request WARNING 2024-06-14 23:23:14,646 log 2369 140528763000384 Not Found: /favicon.ico
django.request WARNING 2024-06-14 23:23:14,701 log 2369 140528754607680 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request ERROR 2024-06-14 23:23:29,851 log 2369 140528746214976 Internal Server Error: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/
Traceback (most recent call last):
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/django-renterprise/orders/views.py", line 69, in order_create
    return render(
           ^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/backends/django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 200, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 513, in parse
    raise self.error(token, e)
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 511, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader_tags.py", line 293, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 513, in parse
    raise self.error(token, e)
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 511, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader_tags.py", line 232, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 513, in parse
    raise self.error(token, e)
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 511, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/defaulttags.py", line 953, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 513, in parse
    raise self.error(token, e)
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 511, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/defaulttags.py", line 1389, in url
    args.append(parser.compile_filter(value))
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 600, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 703, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: '{{' from '{{'
django.request WARNING 2024-06-14 23:23:30,172 log 2369 140528737822272 Not Found: /favicon.ico
django.request WARNING 2024-06-14 23:23:30,185 log 2369 140528729429568 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request ERROR 2024-06-14 23:25:01,323 log 2369 140528242914880 Internal Server Error: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/
Traceback (most recent call last):
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/django-renterprise/orders/views.py", line 69, in order_create
    return render(
           ^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/defaulttags.py", line 321, in render
    return nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/urls/base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/urls/resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'customer_view' with no arguments not found. 1 pattern(s) tried: ['customers/(?P<customer_token>[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})/\\Z']
django.request WARNING 2024-06-14 23:25:01,583 log 2369 140528234522176 Not Found: /favicon.ico
django.request WARNING 2024-06-14 23:25:01,643 log 2369 140528226129472 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request ERROR 2024-06-14 23:26:57,688 log 2369 140528217736768 Internal Server Error: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/
Traceback (most recent call last):
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/django-renterprise/orders/views.py", line 69, in order_create
    return render(
           ^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/defaulttags.py", line 321, in render
    return nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/urls/base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/urls/resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'customer_view' with arguments '('',)' not found. 1 pattern(s) tried: ['customers/(?P<customer_token>[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})/\\Z']
django.request WARNING 2024-06-14 23:26:58,024 log 2369 140528209344064 Not Found: /favicon.ico
django.request WARNING 2024-06-14 23:26:58,040 log 2369 140528200951360 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request ERROR 2024-06-14 23:28:22,443 log 2369 140528192558656 Internal Server Error: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/
Traceback (most recent call last):
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/django-renterprise/orders/views.py", line 69, in order_create
    return render(
           ^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/defaulttags.py", line 321, in render
    return nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/urls/base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/urls/resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'customer_view' with arguments '('',)' not found. 1 pattern(s) tried: ['customers/(?P<customer_token>[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})/\\Z']
django.request WARNING 2024-06-14 23:28:22,705 log 2369 140527706043968 Not Found: /favicon.ico
django.request WARNING 2024-06-14 23:28:22,876 log 2369 140527697651264 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request ERROR 2024-06-14 23:30:30,257 log 2369 140527689258560 Internal Server Error: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/
Traceback (most recent call last):
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/django-renterprise/orders/views.py", line 69, in order_create
    return render(
           ^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/defaulttags.py", line 321, in render
    return nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/urls/base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/urls/resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'customer_view' with keyword arguments '{'customer_token': ''}' not found. 1 pattern(s) tried: ['customers/(?P<customer_token>[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})/\\Z']
django.request WARNING 2024-06-14 23:30:30,526 log 2369 140527680865856 Not Found: /favicon.ico
django.request WARNING 2024-06-14 23:30:30,593 log 2369 140527672473152 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request ERROR 2024-06-14 23:31:24,589 log 2369 140527664080448 Internal Server Error: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/
Traceback (most recent call last):
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/django-renterprise/orders/views.py", line 69, in order_create
    return render(
           ^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/defaulttags.py", line 321, in render
    return nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/urls/base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/urls/resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'customer_view' with arguments '('',)' not found. 1 pattern(s) tried: ['customers/(?P<customer_token>[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})/\\Z']
django.request WARNING 2024-06-14 23:31:24,853 log 2369 140527655687744 Not Found: /favicon.ico
django.request WARNING 2024-06-14 23:31:25,023 log 2369 140527169173056 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request WARNING 2024-06-14 23:33:52,459 log 2369 140527118816832 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request WARNING 2024-06-14 23:34:07,298 log 2369 140526590338624 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.utils.autoreload INFO 2024-06-14 23:36:00,112 autoreload 2369 140538501590080 /workspace/django-renterprise/orders/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 23:36:00,615 autoreload 8950 140027838499904 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-14 23:36:05,873 autoreload 8950 140027838499904 /workspace/django-renterprise/orders/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 23:36:06,352 autoreload 8964 140526236236864 Watching for file changes with StatReloader
django.request ERROR 2024-06-14 23:36:26,018 log 8964 140526095431232 Internal Server Error: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/
Traceback (most recent call last):
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/django-renterprise/orders/views.py", line 73, in order_create
    "customer_token_value": order.customer.customer_token,
                            ^^^^^
UnboundLocalError: cannot access local variable 'order' where it is not associated with a value
django.request WARNING 2024-06-14 23:36:26,241 log 8964 140526087038528 Not Found: /favicon.ico
django.request WARNING 2024-06-14 23:36:26,253 log 8964 140526078645824 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.utils.autoreload INFO 2024-06-14 23:37:10,770 autoreload 8964 140526236236864 /workspace/django-renterprise/orders/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-14 23:37:11,258 autoreload 9061 140201088904256 Watching for file changes with StatReloader
urllib3.connectionpool DEBUG 2024-06-14 23:37:14,259 connectionpool 9061 140200952985152 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-06-14 23:37:14,278 connectionpool 9061 140200952985152 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 200 4117
django.request WARNING 2024-06-14 23:37:16,885 log 9061 140200911021632 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request WARNING 2024-06-14 23:37:42,390 log 9061 140200382543424 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request WARNING 2024-06-14 23:37:46,452 log 9061 140200365758016 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request WARNING 2024-06-14 23:37:46,620 log 9061 140199879243328 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/styles.css.map
django.request WARNING 2024-06-14 23:37:50,881 log 9061 140199862457920 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/styles.css.map
django.request WARNING 2024-06-14 23:37:51,126 log 9061 140199854065216 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request WARNING 2024-06-14 23:37:52,193 log 9061 140199837279808 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request WARNING 2024-06-14 23:37:52,561 log 9061 140199828887104 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/styles.css.map
django.request WARNING 2024-06-14 23:38:53,361 log 9061 140199300408896 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/styles.css.map
django.request WARNING 2024-06-14 23:38:56,276 log 9061 140198805501504 Not Found: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/styles.css.map
django.request WARNING 2024-06-14 23:38:56,481 log 9061 140198788716096 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request WARNING 2024-06-14 23:40:43,134 log 9061 140198260237888 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request WARNING 2024-06-14 23:51:21,611 log 9061 140197731759680 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request WARNING 2024-06-14 23:55:07,811 log 9061 140197681403456 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request WARNING 2024-06-14 23:56:00,717 log 9061 140197152925248 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request WARNING 2024-06-14 23:56:15,582 log 9061 140196624447040 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request WARNING 2024-06-14 23:57:07,121 log 9061 140196095968832 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request WARNING 2024-06-14 23:57:36,718 log 9061 140195567490624 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.utils.autoreload INFO 2024-06-15 00:05:31,157 autoreload 9061 140201088904256 /workspace/django-renterprise/orders/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-15 00:05:31,654 autoreload 10636 140516655004736 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-15 00:16:49,954 autoreload 10636 140516655004736 /workspace/django-renterprise/orders/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-15 00:16:50,434 autoreload 11245 140170832800832 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-15 00:16:53,695 autoreload 11245 140170832800832 /workspace/django-renterprise/orders/views.py changed, reloading.
django.utils.autoreload INFO 2024-06-15 00:16:54,196 autoreload 11257 140590992399424 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-15 00:18:23,866 autoreload 11257 140590992399424 /workspace/django-renterprise/items/admin.py changed, reloading.
django.utils.autoreload INFO 2024-06-15 00:18:24,352 autoreload 11361 140036880997440 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-15 00:18:42,851 autoreload 11361 140036880997440 /workspace/django-renterprise/items/admin.py changed, reloading.
django.utils.autoreload INFO 2024-06-15 00:18:43,339 autoreload 11379 140579027823680 Watching for file changes with StatReloader
urllib3.connectionpool DEBUG 2024-06-15 00:18:49,486 connectionpool 11379 140578962499136 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-06-15 00:18:49,503 connectionpool 11379 140578962499136 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 200 4117
django.request WARNING 2024-06-15 00:18:51,500 log 11379 140578943583808 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.request WARNING 2024-06-15 00:18:52,073 log 11379 140578935174720 Not Found: /orders/8e05b134-0b51-46e4-ac9d-2e725f265a9b/create/styles.css.map
django.utils.autoreload INFO 2024-06-15 00:19:35,458 autoreload 11379 140579027823680 /workspace/django-renterprise/items/admin.py changed, reloading.
django.utils.autoreload INFO 2024-06-15 00:19:35,948 autoreload 11432 139828060580928 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-15 00:26:19,675 autoreload 11432 139828060580928 /workspace/django-renterprise/items/admin.py changed, reloading.
django.utils.autoreload INFO 2024-06-15 00:26:20,171 autoreload 11694 139681229122624 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-15 00:26:24,445 autoreload 11694 139681229122624 /workspace/django-renterprise/items/admin.py changed, reloading.
django.utils.autoreload INFO 2024-06-15 00:26:24,931 autoreload 11715 140417666468928 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-15 00:32:44,349 autoreload 11715 140417666468928 /workspace/django-renterprise/orders/forms.py changed, reloading.
django.utils.autoreload INFO 2024-06-15 00:32:44,831 autoreload 12210 140502561000512 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-15 00:34:01,200 autoreload 12210 140502561000512 /workspace/django-renterprise/orders/models.py changed, reloading.
django.utils.autoreload INFO 2024-06-15 00:34:01,686 autoreload 12304 140204573752384 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-06-15 00:35:06,892 autoreload 12304 140204573752384 /workspace/django-renterprise/orders/forms.py changed, reloading.
django.utils.autoreload INFO 2024-06-15 00:35:07,378 autoreload 12374 140126121524288 Watching for file changes with StatReloader
